scripts/auth.js

```diff
@@ -0,0 +1,25 @@
+//<---------------------------------------------------------------------------------------->
```

Comment:
To avoid confusion between clients-side and server-side scripts, let's avoid including server-side scripts inside the `scripts` directory.

I think we can make a new `modules` directory for all server-side node scripts.